src/js/shared/compare.ts

```diff
@@ -0,0 +1,85 @@
+export function shallowDeepAlmostEqual(
```

Comment:
maybe assert.js file and assert in the name?

Reply:
Done